enterprise/app/root/root.tsx

```diff
@@ -137,33 +142,37 @@ export default class EnterpriseRootComponent extends React.Component {
           </MenuComponent>
         )}
         <div className="page">
-          {sidebar && (
+          {sidebar && !code && (
```

Comment:
nit: edit the `sidebar` bool to include this condition instead?

Reply:
Done